packages/plugins/plugin-table/src/TablePlugin.tsx

```diff
@@ -150,45 +151,26 @@ export const TablePlugin = (): PluginDefinition<TablePluginProvides> => {
               };
             }
 
-            case TableAction.ADD_COLUMN: {
-              const { table, field: _field } = intent.data as TableAction.AddColumn;
-              if (!isTable(table) || !table.view) {
-                return;
-              }
-
-              // TODO(ZaymonFC): We need to manipulate the schema with another method here.
-              // addFieldToView(table.schema, table.view, field);
-              return { data: true };
-            }
-
             case TableAction.DELETE_COLUMN: {
-              const { table, field } = intent.data as TableAction.DeleteColumn;
-              if (!isTable(table) || !table.view) {
-                return;
-              }
+              const { table, property } = intent.data as TableAction.DeleteColumn;
+              invariant(isTable(table));
+              invariant(table.view, 'Table requires a view to delete columns');
 
-              if (!intent.undo) {
-                const fieldPosition = table.view.fields.indexOf(field);
-                if (fieldPosition === undefined) {
-                  return;
-                }
-
-                // TODO(ZaymonFC): We need to manipulate the schema with another method here.
-                // removeFieldFromView(table.schema, table.view, field);
+              const schema = getSpace(table)?.db.schemaRegistry.getSchema(table.view.query.__typename);
+              invariant(schema, 'Table schema not found');
```

Comment:
are these messages helpful? if so, add type at end of message and keep terse.
`Schema not found: ${}`